.github/workflows/pr.yml

```diff
@@ -75,11 +75,25 @@ jobs:
           echo "predict result directory missing" >&2 
           exit 1
         fi
-        echo "result lines "$(wc -l example/results_predict/predict_data_AR.csv)
-        if [ "$(cat example/results_predict/predict_data_AR.csv | wc -l)" -lt "2000" ]; then
-          echo "predict result should have at least 2000 lines. But had only $(cat example/results_predict/predict_data_AR.csv | wc -l)" >&2 
+        echo "result lines "$(wc -l example/results_predict/smiles.csv)
+        if [ "$(cat example/results_predict/smiles.csv | wc -l)" -lt "5" ]; then
+          echo "predict result should have at least 5 lines. But had only $(cat example/results_predict/predict_data_AR.csv | wc -l)" >&2 
+          exit 1
+        fi
+        tree example
+        
+        dfpl traingnn -f example/traingnn.json
+        if [ ! -d dmpnn-random/ ]; then
+          echo "predict result directory missing" >&2 
           exit 1
         fi
         tree example
 
-        dfpl convert -f example/data
\ No newline at end of file
+        dfpl predictgnn -f example/predictgnn.json
+        if [ ! -d preds_dmpnn/ ]; then
+          echo "predict result directory missing" >&2 
+          exit 1
+        fi
+        
```

Comment:
Should we implement a simple test here as well, e.g. check the number of lines in the output like above?